commands.go

```
@@ -426,7 +426,7 @@ func getHostState(host Host, store Store, hostListItems chan<- hostListItem) {
 
 	isActive, err := store.IsActive(&host)
 	if err != nil {
-		log.Errorf("error determining whether host %q is active: %s",
+		log.Debugf("error determining whether host %q is active: %s",
```

Why swallow this error?